modules/images/webp-uploads/load.php

```diff
@@ -185,7 +185,6 @@ function webp_uploads_generate_image_size( $attachment_id, $size, $mime ) {
 
 	$editor->resize( $width, $height, $crop );
```

Comment:
Now that we use the original image, we need to make sure to reapply any rotation based on the `orientation` EXIF data as needed, using the `maybe_exif_rotate()` method. This wasn't needed before when using the scaled or rotated image which already included that, but now we need it.

See the [`_wp_make_subsizes()` function](https://developer.wordpress.org/reference/functions/_wp_make_subsizes/) which also does that.

It's potentially not great to just add that here since it would mean we do this for every sub-size, which would be wasteful, for example the above WP core function does it only once and then generates all subsizes from it.

But for now let's just add it here so that it works, we can separately optimize the performance of that - restructuring the code that much would be a bit too intense for the scope of this PR>

Reply:
Great point, let me add that one.